engine/server.ts

```diff
@@ -185,6 +200,25 @@ export class ChannelEngine {
     if (options && options.useDemuxedAudio) {
       this.useDemuxedAudio = true;
     }
+    this.dummySubtitleEndpoint = dummySubtitleEndpointPath;
+    if (options && options.dummySubtitleEndpoint) {
+      this.dummySubtitleEndpoint = options.dummySubtitleEndpoint;
+    } else {
+      options.dummySubtitleEndpoint = this.dummySubtitleEndpoint;
+    }
+    this.subtitleSliceEndpoint = subtitleSliceEndpointPath;
+    if (options && options.subtitleSliceEndpoint) {
+      this.subtitleSliceEndpoint = options.subtitleSliceEndpoint;
+    } else {
+      options.subtitleSliceEndpoint = this.subtitleSliceEndpoint;
+    }
+    this.shouldContainSubtitles = true;
```

Comment:
Default should be false so we don't break current implementations. Also rename option to be called `useVTTSubtitles`

Reply:
fixed